data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,732 @@
+			choice
+				`	"These colonists are my property by right of conquest. If you're taking them, I expect to be fairly compensated."`
+				`	"Actually, I was just leaving."`
+					goto retreat
+				`	"Whoah, whoah, whoah, I'm cool with forcing people to give me stuff because I'm holding them at gunpoint, but forcing them into slavery? That's sick."`
+					goto challenge
+			`	The chief works his jaw for a moment, deep in thought. "Alright. You soundly thrashed the last bunch of bastards who were here, so I'm inclined to think we're best off doing business. Wholesale rate on these skins is ten thousand a body. 'course, that's with no shipping and handling - factor that in and we can afford to give you ehhh... Quarter million for the lot of 'em. You'll still have some left once we're done, mind."`
+			choice
+				`	"Done deal."`
+			action
+				payment 250000
+			apply
+				set "slaver of Darkstone"
+				set "terribleperson"
+			`	The slave boss counts out an appropriate stack of credits and presses them into your palm. The slaves regard you with a mixture of hatred and resignation. Now rather richer than you would have been just collecting their paltry tribute, you return to the skies.`
+				flee
+			label duel
+			`	The leader of the slavers turns to face you, attention finally pulled from his task. "You've got some nerve picking a fight with us, kid. You really think these poor bastards are worth dying for?"`
+			`	Noon's sun beats down on you. Even with the chill that usually haunts this region of the planet at this time of year, it's enough to bring sweat to your brow. Silence fills the town square, and slavers shy back to leave only four things that matter in the area: You, the slaver boss, and your respective pistols.`
+			`	A bird cries in the distance.`
+			choice
+				`	(Draw!)`
+			`	The slaver boss is fast, but you're faster. You're able to slip your sidearm out of its holster, level it, and cleanly blow him away before he can so much as draw a bead on you - and so it's with the thrill of victory that you die, as you're unceremoniously cut down by massed fire from the gang of slavers surrounding your little duel.`
+			`	The few colonists that are left behind after the slavers have filled their holds honor your sacrifice by rolling your body into a ditch, spitting on it, and then promptly forgetting you ever lived.`
+				die
+			label retreat
+			`	You retreat to your ship, unable or unwilling to defend your extortion-subjects against a slaver incursion.`
+				flee
+			label challenge
+			`	"Yeah? And what do you think you can do about that?"`
+			choice
+				`	"You raise a compelling point, sir, and I wish you a good day while I leave to think further upon that."`
+					goto retreat
+				`	"I think I can kick your ass, that's what."`
+					goto duel
+				`	(Make your excuses and return to your ship. You'll be back - once your engines and weapons are warmed up.)`
```

Comment:
Also here.

Reply:
What are you pointing out in this block?

Reply:
Same thing, your ship could be weaponless